pkg/generators/extension_test.go

```diff
@@ -361,8 +361,17 @@ func TestExtensionKind(t *testing.T) {
 		},
 	}
 	for _, test := range methodTests {
-		if test.kind != test.e.kind() {
-			t.Errorf("%v: kind() expected (%v), but received: %v", test.e, test.kind, test.e.kind())
+		if len(test.kind) > 0 {
```

Comment:
It feels better to add something to the test structure rather than copy-paste the logic we're trying to test. Up to you if you want to fix.